packages/server-side-render/src/server-side-render.js

```diff
@@ -165,6 +170,9 @@ export default function ServerSideRender( props ) {
 					fetchRequest === fetchRequestRef.current
 				) {
 					setIsLoading( false );
+					// Cancel the timeout to show the Spinner.
+					setShowLoader( false );
```

Comment:
Why do we need to `setShowLoader( false )` on cleanup? Feels odd to set the state on unmounting, since we won't be using that state.